src/rm_topk.c

```diff
@@ -308,6 +309,11 @@ static void *TopKRdbLoad(RedisModuleIO *io, int encver) {
         }
     }
 
+    /* Initialize lookupTable */
+    for (uint32_t i = 0; i < TOPK_DECAY_LOOKUP_TABLE; ++i) {
+        topk->lookupTable[i] = pow(topk->decay, i);
```

Comment:
@MeirShpilraien can we build a shared lookup table mechanism to reduce the memory overhead here?

Reply:
We probably can but it is out of the scope of this fix, the goal here is to fix a bug :)